core/src/docs/upgrade.md

```diff
@@ -1,3 +1,11 @@
+# Upgrade to v0.38
+
+## Raw API
+
+OpenDAL add the `Write::sink` API to enable streaming writing. This is a breaking change for users who depend on the raw API.
```

Comment:
Users who implement `opendal::raw::oio::Write` can return `Unsupported` error for `Write::sink()` for a quick fix.